planner/core/testdata/point_get_plan_out.json

```diff
@@ -87,14 +87,14 @@
         "Plan": [
           "Point_Get_1 1.00 root table:t1, clustered index:PRIMARY(id) "
         ],
-        "Res": []
+        "Res": null
```

Comment:
How is this change related to this PR?

Reply:
It's unrelated. But run `go test -record` changes it automatically. Both `[]` and `null` are correct for this test.